.github/workflows/build-fedora.sh

```diff
@@ -318,6 +318,9 @@ for phase in "${PHASES[@]}"; do
             # Setup a local iSCSI target
             target_name="iqn.2022-01.com.example:iscsi.initrd.test"
             pgrep tgtd || /usr/sbin/tgtd
+            # Workaround for a race
+            # See: https://src.fedoraproject.org/rpms/scsi-target-utils/c/3a25fe7a57200b61ecebcec0d867671597080196
+            sleep 5
```

Comment:
Oh, so https://bugzilla.redhat.com/show_bug.cgi?id=848942 was opened against scsi-target-utils in F18, which was "forwarded" to https://bugs.freedesktop.org/show_bug.cgi?id=55431 in the upstream bugtracker for systemd, where Lennart asked a trivial follow-up question and the bug was closed three years later for lack of response. But systemd can hardly be responsible for tgtd not starting properly, so it would have been closed anyway.

So the original bug remains untouched.